tests/annotator/annotator.cpp

```diff
@@ -427,40 +445,24 @@ TEST(Annotator, errorHandling)
     auto model = libcellml::Model::create("model");
     auto annotator = libcellml::Annotator::create();
 
-    // Searching before the map is built should return an error.
-    EXPECT_EQ(libcellml::Annotator::Type::ISSUE, annotator->item("i_dont_exist").first);
-    EXPECT_EQ(size_t(2), annotator->issueCount());
-    EXPECT_EQ(annotator->issue(0)->description(), "No model is stored; no action has been taken.");
-    EXPECT_EQ(annotator->issue(1)->description(), "The model index is out-of-date.  Please (re)build it before attempting to access items by their id.");
```

Comment:
I don't understand why you're not testing which error is returned but only the total?